src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj

```diff
@@ -22,8 +22,9 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.10.0" />
-    <PackageReference Include="Google.Protobuf" Version="3.21.7" />
-    <PackageReference Include="Grpc.Tools" Version="2.49.1">
+    <PackageReference Include="Google.Protobuf" Version="3.23.3" />
+    <PackageReference Include="Grpc.Core" Version="2.46.6" />
```

Comment:
We have a new entry to add `Grpc.Core` with this version bump change?

Reply:
Hmm I don't know what happened there. Let me back that part out.